app/controllers/principals_controller.rb

```diff
@@ -24,29 +26,37 @@ def new
     Stats.increment('radsignup.prequalification.success')
   end
 
-  def show
-  end
+  def show; end
 
   def create
-    @form = NewPrincipalForm.new(new_principal_form_params)
-
-    if @form.valid?
-      user = User.new(@form.user_params)
-      user.build_principal(@form.principal_params)
-      user.save!
+    form = NewPrincipalForm.new(new_principal_form_params)
 
-      Stats.increment('radsignup.principal.created')
-
-      Identification.contact(user.principal).deliver_later
-      redirect_to user.principal
+    if form.valid?
```

Comment:
I'm surprised rubocop didn't suggest a guard clause instead of this if statement.